### PR TITLE
Documentation update for latest Rust SDK

### PR DESCRIPTION
**Pull request by arturkow2000**

### CODE REVIEW

**Review by 0xDen — APPROVED** [review]
Look good to me.
But I need to check this

File: docs/environment.md [hunk 1]
```diff
@@ -17,9 +17,9 @@ In order to build the container follow the steps below:
    git clone https://github.com/fobnail/fobnail-sdk.git
    ```
 3. Go to fobnail SDK directory: `cd fobnail-sdk`
-4. To use latest container checkout `rust` branch: `git checkout rust`
-5. Execute `docker build . -f Dockerfile.rust -t 3mdeb/fobnail-rust-sdk`. It
-   will build the container with following software available:
+4. To use latest SDK version checkout `rust` branch: `git checkout rust`
```
Comment (0xDen) [inline docs/environment.md:20]: ...the latest SDK...

File: docs/flashing_samples.md [hunk 2]
```diff
@@ -9,118 +9,54 @@ Prepare the development environment first as described in
...
-   usb 1-1.4: SerialNumber: 05AA335B46AC1D80
-   cdc_acm 1-1.4:1.0: ttyACM0: USB ACM device
-   ```
-   Notice the different in the Product name: `Open DFU Bootloader` vs
-   `MCUBOOT`. This means we have booted the mcuboot in bootloader mode.
+If `probe-rs` does not detect debugger, that could mean it is not supported. You
+can set `RUST_LOG=trace` environment variable to retry scanning with debug
+output enabled.
+
+For debugger to work, you need to add a Udev rule granting you correct
```
Comment (0xDen) [inline docs/flashing_samples.md:30]: ... `an` Udev rules...

File: docs/environment.md [hunk 3]
```diff
@@ -17,53 +17,27 @@ In order to build the container follow the steps below:
...
 
-1. `git clone https://github.com/fobnail/fobnail.git`
-2. `cd fobnail`
-3. `git submodule update --init --checkout`
-4. `./run-container.sh` This will launch the container. It will take a moment
-   to initialize when invoke for the first time in given directory.
-5. Build a blinky application:
+1. `git clone https://github.com/fobnail/nrf-hal`
+2. `cd nrf-hal`
+3. Start container: `./run-container.sh`
```
Comment (0xDen) [inline docs/environment.md:33]: I would specify a git branch where `run-container.sh` is located. There is no such file in `master` branch

File: docs/environment.md (hunk 1 above)
Comment (macpijan) [reply docs/environment.md:20]: @arturkow2000 Using branches in documentation makes it outdated quickly. I have just merged this. Please skip the checkout part and just use the default branch.

File: docs/flashing_samples.md (hunk 2 above)
Comment (arturkow2000) [reply docs/flashing_samples.md]: fixed

File: docs/environment.md (hunk 3 above)
Comment (arturkow2000) [reply docs/environment.md]: Done

File: docs/environment.md (hunk 1 above)
Comment (arturkow2000) [reply docs/environment.md]: @macpijan Done

**Pull request merged**